docs/python/environments.md

```diff
@@ -4,39 +4,42 @@ Area: python
 TOCTitle: Environments
 ContentId: 8fe4ca8b-fc70-4216-86c7-2c11b6c14cc6
 PageTitle: Using Python Environments in Visual Studio Code
-DateApproved: 10/05/2021
+DateApproved: 03/20/2022
 MetaDescription: Configuring Python Environments in Visual Studio Code
 MetaSocialImage: images/tutorial/social.png
 ---
 # Using Python environments in VS Code
 
-This article discusses the helpful features provided by the VS Code Python extension for working with Python environments. An "environment" in Python is the context in which a Python program runs and consists of an interpreter and any number of installed packages. After you finish this article, you'll have a good understanding of:
+This article discusses the helpful Python environments features available in VS Code. An "environment" in Python is the context in which a Python program runs and consists of an interpreter and any number of installed packages.
 
-- [General environment concepts](#global-and-virtual-environments)
-- [How to select an environment](#select-and-activate-an-environment)
-- How to create a [virtual](#create-a-virtual-environment) or [conda](#create-a-conda-environment) environment
-- How to [select and activate](#select-and-activate-an-environment) a virtual environment
-- How to work with [Environments and Terminal windows](#environments-and-terminal-windows)
-- [Where the Python extension looks for environments](#where-the-extension-looks-for-environments)
-- Environment variables and [environment variable definitions files (.env)](#environment-variable-definitions-file)
+If you're new to working with Python environments, you can learn more at [Getting started](#getting-started).
```

Comment:
I'm concerned if some people will feel like they're being directed out of this doc if they're trying to learn about Python environments, as the getting started section only has links to other docs and tutorials?

Reply:
That's a good point. Maybe the python-tutorial.md located in the Getting Started section at the bottom will be enough. If I notice any feedback from the community, regarding Python environments, I can add links to the bottom later!